doc/classes/DisplayServer.xml

```diff
@@ -665,86 +689,93 @@
 			<return type="bool" />
 			<param index="0" name="feature" type="int" enum="DisplayServer.Feature" />
 			<description>
+				Returns [code]true[/code] if the specified [param feature] is supported by the current [DisplayServer], [code]false[/code] otherwise.
 			</description>
 		</method>
 		<method name="ime_get_selection" qualifiers="const">
 			<return type="Vector2i" />
 			<description>
+				Gets the selected area within the Input Method Editor window.
```

Comment:
It's a text selection in the IME composition string. (`x` is caret position, `y` is selection length).